tests/queryEditor.spec.ts

```diff
@@ -135,3 +135,35 @@ test('"Timeseries custom query" multiple ts OK', async ({ selectors, readProvisi
   const tsWithUnits = tsExternalIds.map((ts, i) => `${ts}-${units[i]}`);
   await expect(panelEditPage.panel.fieldNames).toContainText(tsWithUnits);
 });
+
+
+test('"Event query" as table is OK', async ({ page, gotoDashboardPage, readProvisionedDashboard, grafanaVersion }) => {
+  const dashboard = await readProvisionedDashboard({ fileName: 'weather-station.json' });
+  const dashboardPage = await gotoDashboardPage(dashboard);
+  
+  const panelEditPage = await dashboardPage.gotoPanelEditPage('3')
```

Comment:
what is `3`? can it change in the future? maybe we can make a constant for this?

Reply:
ok!